tock/tock/static/sass/_uswds_theme.scss

```diff
@@ -0,0 +1,43 @@
+// General
+// This corresponds to the variables set in: 
+// https://github.com/uswds/uswds/blob/develop/src/stylesheets/theme/_uswds-theme-general.scss
+
+$theme-image-path:         "../../vendor/uswds_v2.3.1/img";
+
+// Typographic
+// This corresponds to variables set in:
+// https://github.com/uswds/uswds/blob/develop/src/stylesheets/theme/_uswds-theme-typography.scss
+
+$theme-font-path:          "../../vendor/uswds_v2.3.1/fonts";
+$theme-font-type-sans:     "public-sans";
```

Comment:
Probably dumb uwsds question: Aren't these the defaults? If so, why do we need to set them?

Reply:
I think we need to set them here because of how Django digests this, but I'm not totally sure.

Reply:
I assume if we don't set them, they're not set? If so, that's plenty of reason for me!

Reply:
Looks like the default that ships with the system is `$theme-font-path: "../fonts";`, which would only work from within the NPM package. (Which makes sense if you're using the static, generated CSS and not the theme Sass files, I guess. Kinda. Maybe.)

Reply:
This part is interacting oddly with the way we're handling static files for Tock, e.g. path to `external-link.svg` ends up as `/static/css/img/external-link.svg` which doesn't exist.

I'm not sure why `$theme-image-path` isn't being picked up there 😳